src/Components/Aspire_Components_Progress.md

```diff
@@ -7,9 +7,9 @@ As part of the .NET Aspire November preview, we want to include a set of .NET As
 | Npgsql                                  |                  ✅                  |             ✅             |                           ✅                            |              ✅              |          ✅          |        ✅            |         ✅           |              ✅                  |
```

Comment:
I used "blanks" on the components that didn't have metrics after this change to follow the legend, but it seems that all other ones are using a red cross. Let me know what is preferred.

Reply:
Ok. I was confused because it can be done, but not in this repository, it's out of our control. Would you be fine if I change "Requirement can't be met" by "Requirement can't be met without external changes" or something similar that shows that there is nothing preventing from it to happen, just out of this repository's control.